src/resources/views/partials/maincharacter.blade.php

```diff
@@ -0,0 +1,4 @@
+<a href="{{ route('character.view.sheet', ['character_id' => optional($maincharacter)->character_id]) }}">
```

Comment:
As it will lead to an error, we better should not show the link when we don't have any character_id.
Maybe just display either `unknown` or empty field when we don't have a main.